exampleSite/content/docs/configuration/index.md

```diff
@@ -318,12 +318,19 @@ Many of the article defaults here can be overridden on a per article basis by sp
-| -------------------------- | --------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `umamiAnalytics.websiteid` | _Not set_ | The site code generated by Umami Analytics for the website. Refer to the [Analytics docs]({{< ref "partials#analytics" >}}) for more details.                                           |
-| `umamiAnalytics.domain`    | _Not set_ | If using a custom domain with Umami Analytics, provide it here to serve `script.js` from the custom domain.                                                                             |
+| Name                       | Default   | Description                                                                                                                                                                      |
+| -------------------------- |-----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `umamiAnalytics.websiteid` | _Not set_ | The site code generated by Umami Analytics for the website. Refer to the [Analytics docs]({{< ref "partials#analytics" >}}) for more details.                                    |
+| `umamiAnalytics.domain`    | _Not set_ | If using a custom domain with Umami Analytics, provide it here to serve `script.js` from the custom domain.                                                                      |
 | `umamiAnalytics.dataDomains`    | _Not set_ | If you want the tracker to only run on specific domains, provide it for your tracker script. This is a comma delimited list of domain names. Such as "yoursite.com,yoursite2.com". |
+| `umamiAnalytics.enableTrackEvent`    | true      | When set to `true` track event will add automatically. If you do not want to add track event, set it to `false`.                                                            |
 
+### Seline Analytics
+
+| Name                              | Default   | Description                                                                                                                               |
+|-----------------------------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| `selineAnalytics.token`           | _Not set_ | The token generated by Seline Analytics for the website. Refer to the [Analytics docs]({{< ref "partials#analytics" >}}) for more details. |
+| `selineAnalytics.enableTrackEvent` | true      | When set to `true` track event will add automatically. If you do not want to add track event, set it to `false`.                          |
```

Comment:
These should definitely default to false or not set to match all the other analytics engines. Otherwise, you're enabling it for every user by default.

Reply:
> These should definitely default to false or not set to match all the other analytics engines. Otherwise, you're enabling it for every user by default.

In my opinion `enableTrackEvent=true` is recommended for optimal data tracking and display.

If you want to customize event tracking, might set it to `false`, and then add manually invoking `umami.track()` or `seline.track()` for specific events. 
However, for most users this customization is unnecessary.

Reply:
I see, there's an outer guard here in `analytics/main.html`.